go/app/main.go

```diff
@@ -20,7 +20,10 @@ import (
 )
 
 const (
-	ImgDir = "images"
+	ImgDir              = "images"
+	ItemsShemaPath      = "../db/items.db"
+	CategoriesShemaPath = "../db/categories.db"
```

Comment:
[nits]
👮 < タイポ警察です〜！
```suggestion
	ItemsSchemaPath      = "../db/items.db"
	CategoriesSchemaPath = "../db/categories.db"
```

Reply:
タイポ警察めちゃくちゃ助かります〜！ありがとうございます👮🫶
[9c3630b](https://github.com/dongurikoko/mercari-build-training/pull/8/commits/9c3630bf33522134271c3a0427df5ae63a2ffbb6)にて修正しました。